recipe/meta.yaml

```diff
@@ -35,6 +36,6 @@ about:
 
 extra:
   recipe-maintainers:
-    - adbeda
```

Comment:
Did you make this change on purpose?

Reply:
@wolfv perhaps I was not so clear in my initial PR text. :-) 

The reason I suggested editing the maintainer name is that @adbeda is not (no longer?) a GitHub username, whereas @adbedada is (and did work on the feedstock) - so I simply assumed these two usernames relate to the same person. :)

You are right, it is probably better to revert that edit and add me... (Did the latter part already)

Reply:
I removed the maintainer name adbeda(da) completely since I it seems we would otherwise be blocked later, see status on:
https://github.com/conda-forge/dos2unix-feedstock/pull/7